lua/tree-sitter-jinja2.lua

```diff
@@ -1,16 +1,37 @@
 local M = {}
 
+function M.register_parser()
+  local ok, parsers = pcall(require, "nvim-treesitter.parsers")
+  if not ok then
+    return
+  end
 
-local install_info = {
-  url = "https://github.com/geigerzaehler/tree-sitter-jinja2",
-  files = { "src/parser.c" },
-  branch = "main",
-}
+  -- Register the plugin path as a tree-sitter parser to nvim-treesitter.
+  -- This is not required, but it allows nvim-treesitter to manage it e.g. :TS{Install,Update} jinja2
+  local file = debug.getinfo(1).source:match("@(.*/)")
+  local plugin_dir = vim.fn.fnamemodify(file, ":p:h:h:h")
```

Comment:
This is pretty neat. But there is one `:h` too many and `plugin_dir` would point to the parent directory of the plugin.

```suggestion
  local plugin_dir = vim.fn.fnamemodify(file, ":p:h:h")
```